packages/app/src/runner/SpecRunnerHeaderRunMode.vue

```diff
@@ -32,6 +32,7 @@
           <img
             v-if="selectedBrowser.displayName"
             class="min-w-16px w-16px"
+            alt=""
```

Comment:
Might as well treat this image as decorative and hide it from screen readers, the text is right next to it.

Reply:
The empty alt text will mark it as decorative, but if we want to hide it from readers wouldn't we need something like `aria-hidden="true"`?

Reply:
They have the same effect (as would `role="presentation"`), empty `alt` is just more standard for various reasons. It predates all the aria stuff.